pages/voting-guides/index.tsx

```diff
@@ -76,14 +92,40 @@ const VotingGuides: NextPage<{
             {votingGuides && votingGuides.length < 1 && (
               <small>No voting guides</small>
             )}
+
             {votingGuides?.map((guide) => (
               <VotingGuideCard
                 guide={guide as Partial<VotingGuideResult>}
                 key={guide.id}
+                showEdit={user.id === guide.user.id}
               />
             ))}
           </FlagSection>
         </div>
+        {!!savedGuides?.data?.votingGuidesByIds?.length && (
+          <div className={styles.votingContainer}>
+            <FlagSection title="Other Guides">
+              {/* This section uses {election} since we only have one right now.
+                As the app gets more dynamic this must be changed. */}
+              <div className={styles.votingHeader}>
+                <h1>{dateString(election?.electionDate)}</h1>
+                <h2>{election?.title}</h2>
+                <p>{election?.description}</p>
+              </div>
+
+              {savedGuides.isLoading && <LoaderFlag />}
```

Comment:
Just a note, we should look into the feasibility of leveraging react 18 and Suspense for loader flags.

Reply:
Haven't used suspense yet but very keen and curious